### PR TITLE
Add client reqs for default nav structure, improve description

### LINKED ISSUE #1255: Clients SHOULD present first range structure
If the publisher has a multiple top level structure hierarchies (e.g. logical versus physical structures), and has a preference for which the viewer should display by default, then the first hierarchy SHOULD be displayed by the client as a default.

Further evidence towards #1193 plus an additional client requirement on top of that.

### PR DESCRIPTION
**Pull request by azaroth42**
Closes #1255 ;

### CODE REVIEW

File: source/api/presentation/3.0/index.md [hunk 1]
```diff
@@ -767,12 +767,12 @@ The value _MUST_ be an array of objects.
 
 ##### structures
 
-The structure of an object represented as a Manifest can be described using a hierarchy of Ranges.  The top level Ranges of these hierarchies are given in the `structures` property.
+The navigational structure of an object represented as a Manifest can be described using a hierarchy of Ranges. This can be used to describe the "table of contents" of the object or other navigational structures that the user can interact with beyond a simple linear progression described in the Sequence.  The hierarchy is built by nesting the child Range resources in the `items` array of the higher level Range.  The top level Ranges of these hierarchies are given in the `structures` property.
```
Comment (jpstroop) [inline source/api/presentation/3.0/index.md]: I think we should strike the word "navigational", e.g. I have structural metadata that represents the physical structure of a printed book (gatherings, leaves, and sides). I'm not sure that's strictly navigational, or just a structure that can be represented. Also, paging through the sequence is just as much navigation as anything one might do with ranges, so I think the word just fuels confusion.

Comment (azaroth42) [reply source/api/presentation/3.0/index.md]: The main purpose of Ranges is navigation. Agree there are other potential uses of the structure, so happy to move the sentence around a bit, but I think we should keep the notion of navigation as the primary usage

Comment (jpstroop) [reply source/api/presentation/3.0/index.md]: I could turn this into an argument for #1072 but will hold off for now. I think the distinction becomes ever-more blurry w/ A/V

Comment (zimeon) [reply source/api/presentation/3.0/index.md]: Fine to merge after striking the word 'navigational'

Comment (azaroth42) [reply source/api/presentation/3.0/index.md]: :+1: to striking first "navigational". The following sentence describes the expected usage.

**Comment by mikeapp** [comment]
👍

**Comment by azaroth42** [comment]
Merge?

**Pull request merged**